docs/getting-started-guides/kubeadm.md

```diff
@@ -121,104 +138,126 @@ kubeadm to tell it what to do.
 
 Note: Disabling SELinux by running `setenforce 0` is required in order to allow
 containers to access the host filesystem, which is required by pod networks for
-example. You have to do this until kubelet can handle SELinux better.
+example. You have to do this until SELinux support is improved in the kubelet.
```

Comment:
@lukemarsden Weave for example I think ;)
However, we've told everyone to do setenfoce 0 for now, until things are fixed.
I fixed it for Weave in this PR: https://github.com/weaveworks/weave/pull/2801